composer.json

```diff
@@ -5,26 +5,34 @@
   "license": "GPL-2.0",
   "require-dev": {
     "php": ">=7.0",
-    "wp-coding-standards/wpcs": "^1.0.0",
+    "wp-coding-standards/wpcs": "^1.1.0",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
     "wimg/php-compatibility": "^8.1",
     "phpunit/phpunit": "^6",
-    "brain/monkey": "^2"
+    "brain/monkey": "^2",
+    "codedungeon/phpunit-result-printer": "^0.20.3"
+  },
+  "require": {
+    "php": ">=7.0"
   },
   "config": {
     "platform": {
-      "php": "7.0"
-    }
+      "php": "7.1"
```

Comment:
This must not be changed. Having 7.0 in here ensures that all the dependencies respect the theme's oldest compatible PHP version.

Reply:
Won't the `"php": ">=7.0",` line in the `require-dev` ensure this? The 7.0 version will prevent the pretty printer to be used. I mean, we can omit it, I just added it so that running tests locally in the terminal will look better.

Reply:
@dingo-d The `"php": ">=7.0",` in `require-dev` only states that the _minimum_ required version for the project is PHP 7.0. However, when installing dependencies, `composer.json` will still determine their versions based on the PHP version actually in use. Since we want to be backward-compatible to 7.0 though and include `composer.lock` in the repository, we need to "force" the version that they need to be compatible with to 7.0, which is what the `"php": "7.0"` line under `platform` does. Without this, I could run `composer update` locally, installing dependencies based on my local PHP 7.2 version. This would then cause tests to fail on PHP 7.0 for example because Travis-CI runs `composer install`.

I'll look into how we can improve this some time soon, but for now it should remain there to keep the process working.

Reply:
Ok, I'll rewrite this and remove the pretty printer since it won't work on php 7 (the travis failed earlier bacuse of it).

Thanks for the thorough explanation 👍